WalletWasabi.Fluent/Views/Shell/Shell.axaml

```diff
@@ -15,7 +15,27 @@
   </i:Interaction.Behaviors>
   <Panel>
     <shell:Background />
-    <shell:MainScreen x:CompileBindings="False" IsHitTestVisible="{Binding $parent.DataContext.IsMainContentEnabled^, FallbackValue=True}" />
+    <shell:MainScreen x:CompileBindings="False"
+                      IsHitTestVisible="{Binding $parent.DataContext.IsMainContentEnabled^, FallbackValue=True}">
+      <shell:MainScreen.Styles>
+        <Style Selector="shell|MainScreen[IsHitTestVisible=False]">
+          <Style.Animations>
+            <Animation Duration="1:0:0" Delay="0:0:0.25" Easing="{StaticResource FluentEasing}" PlaybackDirection="AlternateReverse" IterationCount="Infinite">
```

Comment:
@wieslawsoltes could you explain how it works?
In my understanding, this should clear and then blur the control continuously. But this is not the case, also I pushed this with a duration of 1 hour, just to show it does not work.

I mean this exactly does what we need now, but it is a mystery for me how.

Reply:
Hey folks! Make sure to use double values for animations:
```
<BlurEffect Radius="10.0" />
```
The animations fail if it tries to animate a int value to a double property sometimes.